@jenkins-cd/js-builder.js

```diff
@@ -141,9 +141,7 @@ function taskTestFast() {
         {
             testMatch: testMatch,
         },
-        {
-            notify: true,
-        },
+        {},
```

Comment:
We might want to do this for all builds, unless we can somehow override the old leaking notifier that occasionally eats up all your ram on MacOS.